src/dockerHubSearch.ts

```diff
@@ -91,43 +91,50 @@ function invokeHubSearch(imageName: string, count: number, cache: boolean): Prom
         cache);
 }
 export interface IHubSearchResponse {
+    /* eslint-disable-next-line camelcase */
     num_pages: number;
```

Comment:
Looks like fixing this warning by rename seems simple.

Reply:
It likely matches the shape of the data coming back from an HTTP request, so I'm not inclined to rename it; unfortunately TS/JS doesn't have attributes that allow us to de-couple the JSON representation from its object representation.